src/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs

```diff
@@ -92,10 +92,14 @@ internal static bool IsEnabled()
                 }
             }
 
-            Task<HttpResponseMessage> responseTask = base.SendAsync(request, cancellationToken);
+            Task<HttpResponseMessage> responseTask = null;
             try
             {
+                responseTask = base.SendAsync(request, cancellationToken);
+
                 await responseTask.ConfigureAwait(false);
+
+                return responseTask.Result;
```

Comment:
This can just be:
```C#
return await responseTask.ConfigureAwait(false);
```

Reply:
Thank you for the feedback, I fixed this.